F2Py.cmake

```diff
@@ -176,14 +176,21 @@ function (f2py_add_module target_name)
     target_compile_definitions(${target_name}
     PRIVATE ${F2PY_ADD_MODULE_COMPILE_DEFS})
   endif()  
-  target_compile_options(${target_name} PRIVATE -cpp)
   # Link dll statically in Windows
   # It can be potentially a PROBLEM!!!
   # But this is the only way found to build a working library on Windows.
   # However, on Linux and MacOS, it throws an error at the linking stage.
   if (WIN32)
     target_link_libraries(${target_name} PUBLIC "-static")
   endif()
+
+  if (APPLE)
+    set_target_properties(${target_name} PROPERTIES
+      BUILD_RPATH "/usr/local/lib"
+      INSTALL_RPATH "/usr/local/lib"    
+    )
```

Comment:
Could you please add a comment why this is needed?

Reply:
Oh, thank you for noticing. I think it is leftover. It doesn't do anything useful. So it will be removed.